### PR TITLE
Use live badge for source repos' users/stars

### PR DESCRIPTION
**Pull request by bl-ue**
Also updates Stackoverflow user count from 9m to 14m. 
See https://stackexchange.com/leagues/1/alltime/stackoverflow

### CODE REVIEW

**Comment by bl-ue** [comment]
Here's a [demo](https://github.com/bl-ue/cheat.sh/tree/patch-1#cheat-sheets-sources), in case you want to see.

File: README.md [hunk 1]
```diff
@@ -822,13 +822,16 @@ all over the world
...
+|Programming languages  |[adambard/learnxinyminutes-docs](https://github.com/adambard/learnxinyminutes-docs)|![](https://img.shields.io/github/contributors-anon/adambard/learnxinyminutes-docs) ![](https://img.shields.io/github/stars/adambard/learnxinyminutes-docs)|Jun 23, 2013|
+|Go                     |[a8m/go-lang-cheat-sheet](https://github.com/a8m/go-lang-cheat-sheet)|![](https://img.shields.io/github/contributors-anon/a8m/go-lang-cheat-sheet) ![](https://img.shields.io/github/stars/a8m/go-lang-cheat-sheet)|Feb 9, 2014|
+|Perl                   |[pkrumnis/perl1line.txt](https://github.com/pkrumins/perl1line.txt)|![](https://img.shields.io/github/contributors-anon/pkrumins/perl1line.txt) ![](https://img.shields.io/github/stars/pkrumins/perl1line.txt)|Nov 4, 2011|
+|Programming languages  |[StackOverflow](https://stackoverflow.com)|[14M](https://stackexchange.com/leagues/1/alltime/stackoverflow)|Sep 15, 2008|
+
+[cheat.sheets users]: https://img.shields.io/github/contributors-anon/chubin/cheat.sheets
+[cheat.sheets stars]: https://img.shields.io/github/stars/chubin/cheat.sheets
```
Comment (abitrolly) [inline README.md:834]: These seem unused.

**Review by abitrolly — COMMENTED** [review]
The `demo` nails it. :D

Just one comment.

**Comment by abitrolly** [comment]
Screenshot for the history.

---

image

File: README.md (hunk 1 above)
Comment (bl-ue) [reply README.md:834]: Ha! You're right. I was testing to see if I could put the links on there one lines to avoid the clutter :)

**Comment by abitrolly** [comment]
Thanks. )

**Comment by chubin** [comment]
Thank you! Very cool and useful improvement. BTW we have some repositories integrated into cheat.sh, which are not listed here, and some repositories that are currently in progress. So the table could be, and should be extended

**Comment by chubin** [comment]
That is how it looks like in mt browsers:
Selection_126

Maybe we should add explicit line break (if nbsp is not possible) between the badges?

**Comment by abitrolly** [comment]
Or it is possible to use unicode symbols and base64 encoded images for logos.


```
https://img.shields.io/github/contributors-anon/chubin/cheat.sheets?label=%F0%9F%91%A5&labelColor=white
```

https://shields.io/

**Comment by chubin** [comment]
Yes, I think it would be better. We also could split this column into two

**Comment by bl-ue** [comment]
Sure!  That's a cool idea.  I'll do it!

**Pull request merged**